README.md

```diff
@@ -99,23 +98,30 @@ cmake --build . --target install
 ```
 TODO: Add simple client and server source code
 
-### Example Programs
-If the `-DBUILD_EXAMPLES=ON` CMake option is used during building,
-example client and server executables will be built and placed
-in the `${CMAKE_BINARY_DIR}/bin` directory.
-Example configuration data is also provided in the `examples/data`
-directory.
+### Running a Test Client and Server Handshake
+If the `-DBUILD_TOOL=ON` CMake option is used during building,
+the XTT tool will be built and placed in the `${CMAKE_BINARY_DIR}/tool` directory.
+
+To create configuration data, run:  
+`./xtt genkey -v root_priv.bin -b root_pub.bin` to create a root key pair.  
+`./xtt genrootcert` to create a root certificate.  
+`./xtt genkey -v server_priv.bin -b server_pub.bin` to create a server key pair.  
+`./xtt genservercert` to create a server certificate.  
```

Comment:
For now, while we still have the server ID, copy the `server_id.bin` file to the `data/server` directory, so it can be used by this command.